mcs/class/corlib/Test/System/StringTest.cs

```diff
@@ -348,15 +348,7 @@ public unsafe void Constructor7_Value_Invalid ()
 	[Test] // ctor (SByte*, Int32, Int32)
 	public unsafe void Constructor7_Value_Null ()
 	{
-		try {
-			new String ((sbyte*) null, 0, 0);
-			Assert.Fail ("#A1");
-		} catch (ArgumentNullException ex) {
-			Assert.AreEqual (typeof (ArgumentNullException), ex.GetType (), "#A2");
-			Assert.IsNull (ex.InnerException, "#A3");
-			Assert.IsNotNull (ex.Message, "#A4");
-			Assert.AreEqual ("value", ex.ParamName, "#A5");
-		}
+		Assert.AreEqual (string.Empty, new String ((sbyte*) null, 0, 0));
```

Comment:
This means we are not expecting an `ArgumentNullException` anymore, correct?

Reply:
correct